docs/reference/output.md

```diff
@@ -74,7 +74,10 @@ Output a function for each method in the conversion interface.
 
 `output:package [PACKAGE][:NAME]` can be defined as
 [CLI argument](./define-settings.md#cli) or [conversion comment](./define-settings.md#conversion).
-Default is `:generated`.
+
+Default is to reuse the same package name from the directory of the file
+specified with `output:file`. However if that lookup fails, the fallback
+is `:generated`.
```

Comment:
Can we make this more specific, that it only works if the output:file is in the same package as the conversion definition?
```suggestion
If `output:file` is in the same directory as the conversion definition,
`output:package` is automatically set. If not, then the default is
`:generated`.
```

Reply:
This assumes it only works when the output is in the same package. I don't think this feature should be limited in that way.